ngx_http_proxy_connect_module/.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
Everything in this directory `ngx_http_proxy_connect_module` is third-party code from https://github.com/chobits/ngx_http_proxy_connect_module. I would rather use a submodule but github cannot generate a zipped release that includes submodule code, so I copy it into this repo directoy.